### PR TITLE
Swift Package Manager and XCFramework support

### PR DESCRIPTION
**Pull request by gersonnoboa**

### CODE REVIEW

File: Example/Podfile [hunk 1]
```diff
@@ -13,6 +13,14 @@ target 'ExampleApp' do
...
   pod 'Kingfisher'
 end
+
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings['BUILD_LIBRARY_FOR_DISTRIBUTION'] = 'YES'
+    end
+  end
+end
```
Comment (gugalo) [inline Example/Podfile:26]: Why is this needed? 
Will this be also required now for the client integration?

File: SalemoveSDK.xcframework/ios-arm64/SalemoveSDK.framework/Headers/SalemoveSDK.h [hunk 2]
```diff
@@ -0,0 +1,11 @@
+//
+
+#import <UIKit/UIKit.h>
```
Comment (gugalo) [inline SalemoveSDK.xcframework/ios-arm64/SalemoveSDK.framework/Headers/SalemoveSDK.h:3]: Are this empty lines needed?

**Review by gugalo — APPROVED** [review]

File: Example/Podfile (hunk 1 above)
Comment (gersonnoboa) [reply Example/Podfile:26]: Yes, it is now needed, because of the XCFramework. That is one of the reasons we need updated documentation.

File: SalemoveSDK.xcframework/ios-arm64/SalemoveSDK.framework/Headers/SalemoveSDK.h (hunk 2 above)
Comment (gersonnoboa) [reply SalemoveSDK.xcframework/ios-arm64/SalemoveSDK.framework/Headers/SalemoveSDK.h:3]: I don't know, but as these files are all autogenerated, I have no real control over them.

**Comment by gersonnoboa** [comment]
!merge
